### DISCUSSION TITLE
Solokey 2 NFC

### DISCUSSION THREAD

**Discussion by d2a4u** [solokeys/solo2#90] [Ideas]
Hi, I emailed support and was told to create an issue here for visibility. When using NFC on solokey 2 with Android, after 2FA, it always redirects the web page to solokeys.com. Is there a way to disable this behaviour? Thanks

**Comment by nickray** [comment]
This isn't a Solo 2 CLI issue, moving into general Solo 2 discussion.

**Comment by felix-gohla** [comment]
> Hi, I emailed support and was told to create an issue here for visibility. When using NFC on solokey 2 with Android, after 2FA, it always redirects the web page to solokeys.com. Is there a way to disable this behaviour? Thanks

Heyo, I think this is hard coded in the ndef app. Have a look at this: <https://github.com/solokeys/solo2/blob/7d1d9ec2bfae231ca4a7263a7e223db9fed51aa5/components/ndef-app/src/ndef.rs#L21-L24>

**Reply by remygillet** [reply]
Got the same problem on iOS

**Reply by felix-gohla** [reply]
@remygillet Could you elaborate on what you mean with "problem"?

**Reply by remygillet** [reply]
Of course : when using the Solo 2 with my iPhone, I put the key on the back of my phone and then, it unlock the 2FA on desired webiste but i still have a notification to visit solokeys website each time...

**Reply by alexjmoore** [reply]
I have just tried to login to Github mobile on Android using NFC as well as Bitwarden and neither work using NFC - the key is read by Android, but it doesn't successfully login, either failing or opening the solo2 website.  When is NFC due to be fixed? Is there any diagnostics I can provide? Running Android 13 on a Pixel 7 Pro

**Comment by coelner** [comment]
within the solo2-cli there should either be an on/off option or a selectable usage like public key url/stub/vcard/fingerprint.
